examples/sdk-examples/src/examples/attributeFilter/index.tsx

```diff
@@ -134,10 +151,19 @@ export const AttributeFilter = (): JSX.Element => (
             between filters.
         </p>
 
-        <p>Dependency implemented via Placeholders.</p>
+        <p>
+            Dependency between parent filter and child filter is implemented via Placeholders and there is
+            preselect value for parent filter demonstrated in this example.
+        </p>
+
+        <p>
+            <strong>
+                Current limitation is that the parent filter must specify attribute elements using their URIs.
+            </strong>
+        </p>
 
         <ExampleWithSource
-            for={AttributeParentChildFilterButtonWithPlaceholder}
+            for={AttributeParentChildFilterButtonWithPlaceholderExample}
```

Comment:
Please also update names for the `source` and `sourceJS` imports so that they are consistent